topo/node/ixia/ixia.go

```diff
@@ -83,6 +92,39 @@ func (n *Node) Create(ctx context.Context) error {
 	return nil
 }
 
+func (n *Node) Status(ctx context.Context) (corev1.PodPhase, error) {
+	ixiaNode := Ixia{}
+	var phase corev1.PodPhase
+	err := error(nil)
+	pod, _ := n.KubeClient.CoreV1().Pods(n.Namespace).Get(ctx, n.Name(), metav1.GetOptions{})
```

Comment:
Is an error return value being ignored here? I'm not sure the signature but the error should be handled in some way and not be ignored

Reply:
Handled error case.